charts/kyverno/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: kyverno
-version: v3.0.0
+version: v3.1.0
```

Comment:
I don't think we need to change the version yet. Let's wait until 1.12 is released before we bump it up.